audformat/core/utils.py

```diff
@@ -1316,12 +1333,12 @@ def read_csv(
 
     starts = None
     if define.IndexField.START in frame.columns:
-        starts = pd.to_timedelta(frame[define.IndexField.START])
+        starts = frame[define.IndexField.START]
```

Comment:
We can drop the call to `pd.to_timedelta()` as the `starts` values are handed to `audformat.segmented_index()`. The same holds for the `ends` values.